### ISSUE TITLE
Are the configurations for the classification correct?

### ISSUE THREAD

**Issue by jongchyisu** [ajbrock/Generative-and-Discriminative-Voxel-Modelling#8]
Hi, I'm trying to run your classification code. Is the current configuration in VRN.py correct? Now the batch size is 1 (which makes it more than 2 hours for training one epoch), and the learning rate is 0.002 and 0.0002 after 12 epochs. What's the batch size and learning rate and training epochs you used in your paper? I couldn't find it in the paper. Thanks very much!

**Comment by ajbrock** [comment]
IIRC (it's been a long time) those config patterns are for inference. Batch size of 50, and a double-up augmentation strategy, each epoch on a Maxwell Titan X took 6+ hours. I don't recall the exact annealing strategy, I think the one I provide should work okay, but I'd strongly recommend using cosine annealing instead, these days.
